### DISCUSSION TITLE
Poisson方程式の対角成分を大きくすると圧力振動が抑えられるのはなぜですか？

### DISCUSSION THREAD

**Discussion by iori3150** [MPS-Basic/MPS-Basic#147] [Q&A]
@Quesys-tech 
後藤本pp.44-48には微弱な圧縮性の導入で圧力の振動が抑えられるとあります。
物理的には粒子配置が柔軟性を持つことで極端な圧力の発生を抑えられるのだろうと理解しているのですが、それで合っていますか？
また、Poisson方程式の対角項が大きくなることで圧力振動が抑えられることは数学的にはどのように説明されるのでしょうか？

**Comment by Quesys-tech** [comment]
> 物理的には粒子配置が柔軟性を持つことで極端な圧力の発生を抑えられるのだろうと理解しているのですが、それで合っていますか？

合っていると思います。

>Poisson方程式の対角項が大きくなることで圧力振動が抑えられることは数学的にはどのように説明されるのでしょうか？

相対的に他の粒子からの影響が小さくなることを意味して圧力振動が抑えられるのではないでしょうか？
